manifests/bunrodeaoutfitter.yaml

```
@@ -0,0 +1,14 @@
+name: Bunrodea Outfitter
+authors: Nova.
+homepage: https://github.com/Nova1422/plugins/tree/main?tab=readme-ov-file#bunrodeaoutfitter
+license: ALL-RIGHTS-RESERVED
+version: v1.0.0-bunrodeaoutfitter
+shortDescription: Adds an outfitter on the Bunrodea planet "Kalbai"
+description: Adds an outfitter on the Bunrodea planet "Kalbai"
+url: https://github.com/Nova1422/plugins/releases/download/v1.0.0-bunrodeaoutfitter/bunrodeaoutfitter.zip
+iconUrl: https://raw.githubusercontent.com/Nova1422/plugins/v1.0.0-bunrodeaoutfitter/myplugins/bunrodeaoutfitter/icon.png
+autoupdate:
+  type: tag
+  regex: ^.*-bunrodeaoutfitter$
+  url: https://github.com/Nova1422/plugins/releases/download/$version/bunrodeaoutfitter.zip
```

if you change `homepage`, you have to use `update_url` to point it at the repo.
```suggestion
  update_url: https://github.com/Nova1422/plugins
  url: https://github.com/Nova1422/plugins/releases/download/$version/bunrodeaoutfitter.zip
```
same for the other manifest